application/controllers/admin/export.php

```diff
@@ -27,6 +27,10 @@ function __construct($controller, $id)
         parent::__construct($controller, $id);
 
         Yii::app()->loadHelper('export');
+
+        Yii::import('application.controllers.admin.*');
+        require_once('printablesurvey.php');
```

Comment:
Using **3rd-Party** Libraries , and there are better solution , and [this](http://www.yiiframework.com/doc/guide/1.1/en/extension.integration#using-namespaced-3rd-party-libraries) work better for 3rd-party